forum.module

```diff
@@ -207,7 +207,7 @@ function forum_node_view($node, $view_mode) {
       if ($parents = taxonomy_term_load_parents_all($node->forum_tid)) {
         $parents = array_reverse($parents);
         foreach ($parents as $parent) {
-          $breadcrumb[] = l($parent->name, 'forum/' . $parent->name);
+	  $breadcrumb[] = l($parent->name, 'forum/'.str_replace(' ', '-',(strtolower($parent->name))));
```

Comment:
Out of the box this module seems to use the term id: `$breadcrumb[] = l($parent->name, 'forum/' . $parent->tid);` and this PR seems to be conflicting with that. So might need to have a condition if the path module is enabled.